SHUTDOWN_ORDER_FIX.md

```
@@ -0,0 +1,310 @@
+close(oc.stopChan)
+oc.cancelableCtx.Cancel()
+```
+
+**Effect:**
+- All pod handlers receive shutdown signal
+- New pods stop being enqueued
+- In-flight handlers start winding down
+- Controllers stop generating new work
+
+**Timeline:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)